EIPS/eip-x.md

```diff
@@ -127,19 +135,19 @@ MUST NOT revert unless due to integer overflow caused by an unreasonably large i
 
 #### requestRedeem
 
-Assumes control of `shares` from `owner` and submits a request for asynchronous `redeem/withdraw`. This places the Request in Pending state, with a corresponding increase in `pendingRedeemRequest` for the amount `shares`. 
+Assumes control of `shares` from `owner` and submits a Request for asynchronous `redeem/withdraw`. This places the Request in Pending state, with a corresponding increase in `pendingRedeemRequest` for the amount `shares`. 
 
-MAY support either a locking or a burning mechanism for `shares` depending on the vault implemention. 
+MAY support either a locking or a burning mechanism for `shares` depending on the Vault implemention. 
 
-MUST support a redeem Request flow where the `shares` control of shares is taken from `owner` directly where `msg.sender` has EIP-20 approval over the `shares` of `owner`.
+If a Vault uses a locking mechanism for `shares`, those `shares` MUST be burned from the Vault balance before or upon claiming the Request.
 
-When the Request is claimable, `maxRedeem` and `maxWithdraw` will be increased, and `redeem` or `withdraw` can be called by `operator` to receive `assets`. A Request MAY transition straight to claimable state but MUST NOT skip the claimable state.
+MUST support a redeem Request flow where the control of `shares` is taken from `owner` directly where `msg.sender` has EIP-20 approval over the `shares` of `owner`.
 
-If a Vault uses a locking mechanism for `shares`, those `shares` MUST be burned from the vault balance before or upon claiming the Request.
+When the Request is Claimable, `maxRedeem` and `maxWithdraw` will be increased for the case where the `owner` input is the `operator`. `redeem` or `withdraw` can subsequently be called by `operator` to receive `assets`. A Request MAY transition straight to Claimable state but MUST NOT skip the Claimable state.
```

Comment:
same comment as above on skipping to claimable state (realize that it was present in the prior spec as well, think i'm missing something)

Reply:
see above reply